gitmanager.py

```diff
@@ -148,13 +130,13 @@ def add_to_blacklist(cls, **kwargs):
                        "-m", u"Auto {0} of {1} by {2} --autopull".format(op, item, username))
 
             if code_permissions:
-                git.checkout("master")
-                git.merge(branch)
```

Comment:
So we're letting GH handle *all* of the commits then?  We had this here because if you have code privs you should already push this but also *locally* merge it.  As you refactor it, this will prevent in-place activation of changed things if one has code perms.

Reply:
This also was built to push to master directly, hence why we checked *out* master to begin with.  your changes will fubar this process.

Reply:
This is a valid point, however I'm inclined to prefer that we wait for CI.